tests/Tests/500__jupyterhub/custom-image.robot

```diff
@@ -0,0 +1,43 @@
+*** Settings ***
+Documentation    Testing the backend for custom images (Adding ImageStream to redhat-ods-applications)
+Resource         ../../Resources/ODS.robot
+Resource         ../../Resources/Common.robot
+Resource         ../../Resources/Page/ODH/JupyterHub/JupyterHubSpawner.robot
+Resource         ../../Resources/Page/ODH/JupyterHub/JupyterLabLauncher.robot
+Resource         ../../Resources/Page/ODH/JupyterHub/GPU.resource
+Library          JupyterLibrary
+Library          OpenShiftCLI
+Suite Teardown   Custom Image Teardown
+
+
+*** Variables ***
+${YAML} =         configs/resources/custom_image.yaml
+
+
+*** Test Cases ***
+Verify ImageStream Can Be Created
+    [Documentation]    Applies the YAML and Gets the ImageStream
+    [Tags]    Sanity
+    ...       ODS-1208
+    ${apply_status} =    Run Keyword And Return Status    Run    oc apply -f ${YAML}
```

Comment:
If possible could we also change the variable name?

Reply:
Sure